src/app/tests/suites/OTA_SuccessfulTransfer.yaml

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Test OTA Software Update Successful Transfer
+
+config:
+    nodeId: 0x12344321
+    endpoint: 0
+    providerNodeId:
+        type: NODE_ID
+        defaultValue: 0xC0FFEE
+    providerPayload:
+        type: CHAR_STRING
+        defaultValue: "MT:-24J0IX4122-.548G00" # This value needs to be generated
+    otaImageFilePath:
+        type: CHAR_STRING
+        defaultValue: "/tmp/otaImage"
+    rawImageFilePath:
+        type: CHAR_STRING
+        defaultValue: "/tmp/rawImage"
+    rawImageContent:
+        type: CHAR_STRING
+        defaultValue: "Have a hootenanny!"
+    downloadImageFilePath:
+        type: CHAR_STRING
+        defaultValue: "/tmp/downloadedImage"
+
+tests:
+    - label: "Create OTA image"
+      cluster: "SystemCommands"
+      command: "CreateOtaImage"
+      arguments:
+          values:
+              - name: "otaImageFilePath"
+                value: otaImageFilePath
+              - name: "rawImageFilePath"
+                value: rawImageFilePath
+              - name: "rawImageContent"
+                value: rawImageContent
+
+    - label: "Start the provider with an image"
+      cluster: "SystemCommands"
+      command: "Start"
+      arguments:
+          values:
+              - name: "registerKey"
+                value: "chip-ota-provider-app"
+              - name: "discriminator"
+                value: 50
+              - name: "port"
+                value: 5560
```

Comment:
Should these two values be config variables for the test, defaulting to those values?

Reply:
I will turn them into config variables for all the params to ota-provider-app